### PR TITLE
Add fixed fee per ticket

### PR DESCRIPTION
**Pull request by Tho85**

### CODE REVIEW

File: pretix_servicefees/views.py [hunk 1]
```diff
@@ -5,23 +5,33 @@
...
-    service_fee_abs_resellers = forms.DecimalField(label=_('Service fee with resellers'))
+    service_fee_per_ticket = forms.DecimalField(
+        label=_('Fixed fee per ticket'),
+        help_text=_('This fee will be added for each ticket sold, except for free items and addons.')
+    )
+
+    service_fee_abs_resellers = forms.DecimalField(label=_('Fixed fee per order'))
     service_fee_percent_resellers = forms.DecimalField(
-        label=_('Service fee with resellers (%)'),
+        label=_('Variable fee per order'),
```
Comment (raphaelm) [inline pretix_servicefees/views.py]: I think "variable" sounds wrong, shall we say "Percentual fee" maybe?

**Review by raphaelm — COMMENTED** [review]
Sorry this took me so long to take a look at! Looks like good work, I haven't tested it in detail, though. I have one comment on naming. Also, can you reset all of the locale files? It makes the diff hard to review. I'm happy to re-generate them and upload them to translate.pretix.eu after the merge.

File: pretix_servicefees/views.py (hunk 1 above)
Comment (Tho85) [reply pretix_servicefees/views.py]: Yeah, I was looking for a better translation as well. It looks like there's no English word for the opposite of a "fixed fee". Let's settle with "Percentual fee". :+1:

**Comment by Tho85** [comment]
I have updated this PR and left out the locales, please have a look.

**Pull request merged**